source/stylesheets/modules/_app-pane.scss

```diff
@@ -26,6 +26,7 @@
       .app-pane__body {
         display: flex;
         flex: 1 1 100%;
+        min-height:0;
```

Comment:
Yep. Good spot. We should probably look at getting the linter hooked up to pick up on this sort of thing.